.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes #(issue)
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## Updated user documentation:
+
+Please insert user documentation that should be updated (as in the wiki).
+
+See [CONTRIBUTING.md User Documentation section](../CONTRIBUTING.md#user-documentation) for further details.
+
+**Note: Manual page changes must be performed in a commit, not in this PR section.**
+
+# Checklist:
+
+- [ ] Tests pass with `cargo test --all-targets --all-features`
+- [ ] Code is formatted with `cargo fmt -- --check`
+- [ ] Code is reviewed with `cargo clippy --release`
```

Comment:
Those three lines feel kind of redundant to the CI checks, though for completeness they also make sense.